.travis.yml

```diff
@@ -41,7 +40,7 @@ matrix:
 install:
   - if [[ "$TRAVIS_OS_NAME" == "osx"   ]]; then ./script/travis-install-macos; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ./script/travis-install-linux; fi
-  - if [[ "$PODSPEC" ]]; then rvm system; sudo gem install bundler; bundle install; fi
```

Comment:
Using the system Ruby failed to lint the podspec: https://travis-ci.org/Quick/Quick/jobs/365026703

```
-> Quick (1.2.0)
    - WARN  | url: There was a problem validating the URL https://github.com/Quick/Quick.
```